platforms/ios/HelloCordova/Plugins/WizCanvasPlugin/WizCanvasView.m

```diff
@@ -235,13 +235,18 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)orientati
 #pragma mark Script loading and execution
 
 - (BOOL)loadRequest:(NSString *)url {
-    NSLog(@"Loading Script: %@", url );
+    NSLog(@"Loading Script: %@", url);
 
-    NSError *error;
+    NSError *error = nil;
     NSData *urlData = [NSData dataWithContentsOfURL:[NSURL URLWithString:url] options:0 error:&error];
 
-    if (!urlData || error) {
-        NSLog(@"Error: Can't Find Script %@", url );
+    if (error) {
+        NSLog(@"Error: Could not load the requested url %@: %@", url, [error localizedDescription]);
```

Comment:
I'm not sure why we should drop it. The error description might not always be useful but it doesn't seem bad per se.